### PR TITLE
Feature/utm format

### PR DESCRIPTION
**Pull request by j8seangel**
This PR includes UTM as a new coordinate formate available on settings.

And include a helper to get the coordinates depending on the format selected, previously it was done in two places separately (map header and alert distance)

Also includes some common types.

untitled

Bonus: use the version number from reducer and use grey text styles.

### CODE REVIEW

File: app/components/login/styles.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default StyleSheet.create({
   },
   versionText: {
     fontFamily: Theme.font,
-    color: Theme.fontColors.main,
+    color: Theme.fontColors.light,
```
Comment (sorodrigo) [inline app/components/login/styles.js]: You discussed this with JC?

File: app/types/helpers/map.types.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export type Coordinates = {
```
Comment (sorodrigo) [inline app/types/helpers/map.types.js]: I’m not sure, I don’t want this to transform into the constants nightmare in fw desktop.

What do you think of having a `common.types` file?

These types aren’t helpers specific, also used in some components.

Comment (j8seangel) [reply app/types/helpers/map.types.js]: This name sounds better, I'll move to it.

File: app/components/login/styles.js (hunk 1 above)
Comment (j8seangel) [reply app/components/login/styles.js]: Yes, I did.

**Pull request merged**